### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by andrepcg** [nicksp/redux-webpack-es6-boilerplate#6]
Running on windows gives the following error:

`'rm' is not recognized as an internal or external command`

**Comment by nicksp** [comment]
`rm` command is Unix specific. That's why it's failing on Win.
Will look into what can i do. Thanks for notifying me @andrepcg.

**Comment by andrepcg** [comment]
Maybe creating a small node script to remove and create the directory and instead of directly calling the command in the `clean:build` or `prestart` script you just call the node script.

Something like `clean.js`

```
var rimraf = require('rimraf');
var fs = require('fs');

var dir = '../build';

rimraf(dir, function(err){
  if(err) throw err;

  if(process.argv.indexOf("createdir") >= 0)
    fs.mkdirSync(dir);
});
```

And in `package.json`

```
{
  "clean:build": "node ./bin/clean.js createdir",
  "prestart": "node ./bin/clean.js"
}
```

Requires `rimraf` package.

**Comment by nicksp** [comment]
Fixed by #9.
